internal/dao/types.go

```diff
@@ -176,3 +176,14 @@ type Valuer interface {
 	// GetValues returns values for a resource.
 	GetValues(path string, allValues bool) ([]byte, error)
 }
+
+type OwnerInfo struct {
```

Comment:
nit: Docs? + we should probably use `ControllerInfo` vs `OwnerInfo`

Reply:
docs added

Reply:
You are correct. I think OwnerInfo is the way to go given the controller option.